ui/components/app/permission-page-container/permission-page-container.component.js

```diff
@@ -73,52 +52,11 @@ export default class PermissionPageContainer extends Component {
 
   getRequestedMethodState(methodNames) {
     return methodNames.reduce((acc, methodName) => {
-      ///: BEGIN:ONLY_INCLUDE_IN(snaps)
-      if (methodName === RestrictedMethods.wallet_snap) {
-        acc[methodName] = this.getDedupedSnapPermissions();
```

Comment:
Do we not need all of this to be adde back in case of permission requests that have both snaps and other permissions?

Reply:
ah right, good catch.

Reply:
Keeping the deduped permission logic out but bringing back the installation privacy warning. Multiple `wallet_requestPermissions` calls would trigger the connect flow and leave an empty screen for already installed snaps. Instead of coming up with a new screen in the short amount of time we have, we though it best to just allow for seeing the connection screen again.